sklearn/decomposition/_kernel_pca.py

```diff
@@ -138,12 +176,22 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         and Klaus-Robert Mueller. 1999. Kernel principal
         component analysis. In Advances in kernel methods,
         MIT Press, Cambridge, MA, USA 327-352.
+
+    For eigen_solver == 'arpack', refer to `scipy.sparse.linalg.svds`.
+
+    For eigen_solver == 'randomized', see:
```

Comment:
It'd be even better to have these as references in the UG instead of here. See e.g. the UG on SGD https://scikit-learn.org/dev/modules/sgd.html

Reply:
done. good suggestion !